src/panel/app_panel.js

```diff
@@ -62,8 +67,6 @@ export default class AppPanel {
 
     this.menu = new Menu();
 
-    this.activePoiId = null;
```

Comment:
this is still used in `poi_popup.js` (it is supposed to avoid showing the popup on click)

Reply:
Indeed, I broke this behavior. I'll repair it.

Reply:
This is fixed with this commit https://github.com/QwantResearch/erdapfel/pull/322/commits/471d935cab3985f41d6bb833a0f775a221848f6e